ccm_web/package.json

```diff
@@ -10,7 +10,6 @@
     "build:client": "webpack --config webpack/webpack.prod.ts",
     "build:server": "tsc --project tsconfig.build.json",
     "preprod": "npm run build:client && npm run build:server",
-    "prod": "NODE_ENV=production node dist/server/src/main.js",
```

Comment:
I think there's a line in the README about this. We should remove it if you want to wipe this out (which I think is fine).

Reply:
Done [`bfa4929` (#125)](https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/125/commits/bfa4929fc93af2504b3d05abe82d631dc6a602f4)